distsql/stream.go

```diff
@@ -123,10 +123,9 @@ func (r *streamResult) flushToChunk(chk *chunk.Chunk) (err error) {
 			}
 		}
 	}
+	r.curr.RowsData = remainRowsData
```

Comment:
why changing this?

Reply:
For improving the test coverage.
Before this, the else branch will not be touched.